frontend/src/components/Body/BodyOne/BodyOne.js

```diff
@@ -0,0 +1,45 @@
+// LOGO
+// IMG / BANNER
+// Description & image
+
+import './BodyOne.css'
+
+function BodyOne() {
+    return (
+        <div className="BodyOne">
+            <header className="Logo">
+                <img src="" alt="[ LOGO ]" />
```

Comment:
i love that you thought to add alt text to your images! this will go a long way with accessibility concerns in the future

however, when it comes to alt text itself, it's important to know that this is the text that will be read aloud by screen readers and also indexed by search engines. i know this is just a placeholder, but commenting this now so you know what to replace later

also in general, if you have placeholder work, it's a decently good practice to add a comment such as the following:

`// TODO (T300): replace with correct images`

where T300 represents the unique ticket number in your project tracker. this implies also that it is good practice to track these tasks down in your task tracker!